gateways/ssm_direct.sh

```diff
@@ -0,0 +1,5 @@
+aws ssm start-session \
+    --target $TUNNEL_GATEWAY_HOST \
+    --document-name AWS-StartPortForwardingSessionToRemoteHost \
+    --parameters "{\"host\":[\"$TUNNEL_TARGET_HOST\"], \"portNumber\":[\"$TUNNEL_TARGET_PORT\"], \"localPortNumber\":[\"$TUNNEL_LOCAL_PORT\"]}" &
```

Comment:
```suggestion
aws ssm start-session $TUNNEL_SSM_OPTIONS \
    --target $TUNNEL_GATEWAY_HOST \
    --document-name AWS-StartPortForwardingSessionToRemoteHost \
    --parameters "{\"host\":[\"$TUNNEL_TARGET_HOST\"], \"portNumber\":[\"$TUNNEL_TARGET_PORT\"], \"localPortNumber\":[\"$TUNNEL_LOCAL_PORT\"]}" &
```
I just hit an issue with this opening a connection to an instance in us-west-2 while doing a deployment in us-east-1. It would be great to include support for the `ssm_options` like the current ssm tunnel does so this works cross-region. just setting the AWS_REGION environment variable isn't working for some reason...